php/prompt-chatgpt/src/index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+require(__DIR__ . '/../vendor/autoload.php');
+
+return function ($context) {
+    throw_if_missing($_ENV, ['OPENAI_API_KEY']);
+
+    if ($context->req->method === 'GET') {
+        return $context->res->send(get_static_file('index.html'), 200, [
+            'Content-Type' => 'text/html',
```

Comment:
In some PRs you do this, but in some I think you also specify UTF8 charset. Let's make it consistent across all pls.